src/serac/physics/heat_transfer.hpp

```diff
@@ -440,8 +430,8 @@ class HeatTransfer<order, dim, Parameters<parameter_space...>, std::integer_sequ
   {
     ThermalMaterialIntegrand<MaterialType> integrand(material);
 
-    residual_->AddDomainIntegral(Dimension<dim>{}, DependsOn<0, 1, NUM_STATE_VARS + active_parameters...>{}, integrand,
-                                 mesh_);
+    residual_->AddDomainIntegral(Dimension<dim>{}, DependsOn<0, 1, NUM_STATE_VARS + active_parameters...>{},
+                                 std::move(integrand), mesh_);
```

Comment:
It would be better to just create the object in place (you will get guaranteed copy elision in c++17) instead of creating it, then calling move.